package.json

```diff
@@ -100,7 +100,7 @@
     "cozy-scripts": "^8.3.0",
     "cozy-sharing": "12.3.0",
     "cozy-stack-client": "^45.2.0",
```

Comment:
Can you update also `cozy-stack-client` into `46.1.0` so all client packages are at the latest version ? 🙏